workflow-examples/src/test/java/com/redhat/parodos/examples/tekton/TektonWorkFlow.java

```diff
@@ -0,0 +1,164 @@
+package com.redhat.parodos.examples.tekton;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.api.WorkflowDefinitionApi;
+import com.redhat.parodos.sdk.invoker.ApiClient;
+import com.redhat.parodos.sdk.invoker.Configuration;
+import com.redhat.parodos.sdk.model.ArgumentRequestDTO;
+import com.redhat.parodos.sdk.model.ProjectResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowDefinitionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowExecutionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowRequestDTO;
+import com.redhat.parodos.sdk.model.WorkRequestDTO;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import org.junit.Test;
+
+import org.springframework.http.HttpHeaders;
+
+import static com.redhat.parodos.sdkutils.WorkFlowServiceUtils.getProjectAsync;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public class TektonWorkFlow {
+
+	private final String projectName = "project-1";
+
+	private final String projectDescription = "Tekton example project";
+
+	private final String workflowName = "kubeapiWorkFlow";
+
+	private final String taskName = "kubeapiTask";
+
+	@Test
+	public void runFlow() {
+		ApiClient defaultClient = Configuration.getDefaultApiClient();
+
+		defaultClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+
+		try {
+			ProjectResponseDTO testProject = getProjectAsync(defaultClient, projectName, projectDescription);
+
+			// GET workflow DEFINITIONS
+			WorkflowDefinitionApi workflowDefinitionApi = new WorkflowDefinitionApi(defaultClient);
+			List<WorkFlowDefinitionResponseDTO> simpleSequentialWorkFlowDefinitions = workflowDefinitionApi
+					.getWorkFlowDefinitions(workflowName);
+			assertEquals(1, simpleSequentialWorkFlowDefinitions.size());
+
+			// GET WORKFLOW DEFINITION BY Id
+			WorkFlowDefinitionResponseDTO simpleSequentialWorkFlowDefinition = workflowDefinitionApi
+					.getWorkFlowDefinitionById(simpleSequentialWorkFlowDefinitions.get(0).getId());
+
+			// EXECUTE WORKFLOW
+			WorkflowApi workflowApi = new WorkflowApi();
+
+			// 1 - Define WorkRequests
+			String kubeconfigJson = Files.readString(new File("kubeconfig.json").toPath());
+
```

Comment:
this is hard to view. you can use a similar way as in prebuilt-tasks/src/test/java/com/redhat/parodos/tasks/deploy/DeployApplicationTaskTest.java to define the json as:
```
			// @formatter:off
			String deploymentContent = """
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-deployment
spec:
  replicas: 3
  selector:
    matchLabels:
      app: my-app
  template:
    metadata:
      labels:
        app: my-app
    spec:
      containers:
      - name: my-container
        image: my-image:latest""";
			// @formatter:on
```

Reply:
I will try

Reply:
fixed